src/main/resources/testnet.conf

```diff
@@ -50,6 +50,22 @@ ergo {
       version2ActivationDifficultyHex = "20"
     }
 
+    reemission {
+      checkReemissionRules = true
+
+      emissionNftId = "06f29034fb69b23d519f84c4811a19694b8cdc2ce076147aaa050276f0b840f4"
+
+      reemissionTokenId = "01345f0ed87b74008d1c46aefd3e7ad6ee5909a2324f2899031cdfee3cc1e022"
+
+      reemissionNftId = "06f2c3adfe52304543f7b623cc3fccddc0174a7db52452fef8e589adacdfdfee"
+
+      activationHeight = 188001
+
+      reemissionStartHeight = 186400
+
+      injectionBoxBytesEncoded = "a0f9e1b5fb011003040005808098f4e9b5ca6a0402d1ed91c1b2a4730000730193c5a7c5b2a4730200f6ac0b0201345f0ed87b74008d1c46aefd3e7ad6ee5909a2324f2899031cdfee3cc1e02280808cfaf49aa53506f29034fb69b23d519f84c4811a19694b8cdc2ce076147aaa050276f0b840f40100325c3679e7e0e2f683e4a382aa74c2c1cb989bb6ad6a1d4b1c5a021d7b410d0f00"
```

Comment:
Same question about ErgoTree version change to v1.

Reply:
This box is even spent in the testnet atm